src/main/java/frc/robot/subsystems/Elevator/Elevator.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.subsystems.Elevator;
+
+import static frc.robot.Constants.ROBOT_MODE;
+
+import org.littletonrobotics.junction.Logger;
+import frc.robot.pioneersLib.subsystem.Subsystem;
+
+public class Elevator extends Subsystem<ElevatorStates> {
+    private static Elevator instance;
+
+    private ElevatorIO io;
+    private ElevatorIOInputsAutoLogged inputs;
+
+    private Elevator() {
+        super("Elevator", ElevatorStates.IDLE);
+        this.io = switch (ROBOT_MODE) {
+            case SIM -> new ElevatorIOSim();
+            default -> new ElevatorIOSim();
```

Comment:
??? why is default sim an no IO for real or replay (if replay isn't in constants add it)